PREPEND-README.md

```diff
@@ -0,0 +1,42 @@
+# Bash hook to prepend branch name
+This explains how to use and setup a git hook that will prepend the issue number
+to a commit message. Git hooks are used to automate git commands and functions. There are a number of different options, but this particular hook will add the issue number to your "git commit-m" message once it is pushed to the repository. Git hooks live in a local git/hooks file of each repo, so initializing this hook in the robotics-prototype repo will not change any other repos you might have.
```

Comment:
please use back ticks (`) for formatting commands to make them standout/more readable. So instead of "git commit -m" `git commit -m`